signac_dashboard/modules/video_viewer.py

```diff
@@ -50,12 +50,14 @@ def __init__(
         name="Video Viewer",
         context="JobContext",
         template="cards/video_viewer.html",
-        video_globs=["*.mp4", "*.m4v"],
+        video_globs=None,
```

Comment:
Same as above, use a tuple for an immutable argument.
```suggestion
        video_globs=("*.mp4", "*.m4v"),
```